### PR TITLE
Attempt to fix user snapshotter.

### PR DESCRIPTION
**Pull request by bweston92**

### CODE REVIEW

**Review by prolic — COMMENTED** [review]
LGTM

Did you check it's working?

File: service/user-write/bin/user_snapshotter.php [hunk 1]
```diff
@@ -13,32 +13,39 @@
...
-
+$pdoConnection = $factories['pdoConnection']();
 $aggregateDefinition = new UserAggregateDefinition();
+$mongoClient = $factories['mongoConnection']()->client();
+
+/* @var EventStore $eventStore */
 
 $readModel = new SnapshotReadModel(
     $factories['snapshotStore'](),
     new UserAggregateDefinition()
```
Comment (basz) [inline service/user-write/bin/user_snapshotter.php:35]: how does this consume UserAggregateDefinition https://github.com/prooph/snapshotter/blob/develop/src/SnapshotReadModel.php#L54-L56

Comment (prolic) [reply service/user-write/bin/user_snapshotter.php:35]: You're right. This code cannot work.

Comment (bweston92) [reply service/user-write/bin/user_snapshotter.php:35]: @basz different `SnapshotReadModel` https://github.com/prooph/micro/blob/master/src/SnapshotReadModel.php#L42

**Comment by bweston92** [comment]
@prolic I have it working yes.

File: service/user-write/bin/user_snapshotter.php (hunk 1 above)
Comment (prolic) [reply service/user-write/bin/user_snapshotter.php:35]: You're right

**Pull request merged**